src/pages/stacks-blockchain/overview.md

```diff
@@ -20,62 +20,24 @@ replica as all other peers.
 _Non-enumerable_ means that the set of peers that are producing the blocks don’t know about one another — they don’t know
 their identities, or even how many exist and are online. They are indistinguishable.
 
-## Roadmap
+## Tutorials
 
-Stacks improvement proposals (SIPs) are aimed at describing the implementation of the Stacks blockchain, as well as
-proposing improvements. They should contain concise technical specifications of features or standards and the rationale
-behind it. SIPs are intended to be the primary medium for proposing new features, for collecting community input on a
-system-wide issue, and for documenting design decisions.
-
--> See [SIP 000](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-000-stacks-improvement-proposal-process.md) for more details about the SIPs process.
-
-- ✅ [SIP 001:](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-001-burn-election.md) Burn Election
-- ✅ [SIP 002:](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-002-smart-contract-language.md) Clarity, a language for predictable smart contracts
-- ✅ [SIP 003:](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-003-peer-network.md) Peer Network
-- ✅ [SIP 004:](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-004-materialized-view.md) Cryptographic Committment to Materialized Views
-- ✅ [SIP 005:](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-005-blocks-and-transactions.md) Blocks, Transactions, and Accounts
-- [ ] SIP 006: Clarity Execution Cost Assessment _(Q2 2020)_
-- [ ] SIP 007: Stacking Consensus _(Q2 2020)_
-
-## Testnet phases
-
-- ✅ **Helium** is a developer local setup, mono-node, assembling SIP 001, SIP 002, SIP 004 and SIP 005. With this version, developers can not only run Stacks 2.0 on their development machines, but also write, execute, and test smart contracts. See the instructions below for more details.
-- [ ] **Neon** is the upcoming version of our public testnet, that we're anticipating will ship in Q2 2020. This testnet will ship with SIP 003, and will be an open-membership public network, where participants will be able to validate and participate in mining testnet blocks.
-- [ ] **Mainnet** is the fully functional version, that we're intending to ship in Q3 2020.
-
-## Features
-
-✅ Live features&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<input type="checkbox" disabled /> Upcoming
+[@page-reference | grid-small]
+| /stacks-blockchain/managing-accounts, /stacks-blockchain/sending-tokens, /stacks-blockchain/running-testnet-node
```

Comment:
This is actually the same content wrapped in a tutorial format. I agree that this will likely be linked to from different places. I'll add a reference whenever I'll come across content that would benefit from it